graphql/Dockerfile

```diff
@@ -1,60 +1,64 @@
 # ===============================================
-FROM helsinkitest/node:16-slim AS appbase
+FROM registry.access.redhat.com/ubi10/nodejs-22 AS appbase
 # ===============================================
+# install yarn
+USER root
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+RUN yum -y install yarn
+
+WORKDIR /app
 
 # Offical image has npm log verbosity as info. More info - https://github.com/nodejs/docker-node#verbosity
 ENV NPM_CONFIG_LOGLEVEL warn
 
-ARG DEBUG
-ENV DEBUG $DEBUG
-
-# header size issue
-ENV NODE_OPTIONS=--max-http-header-size=16384
+# set our node environment, either development or production
+# defaults to production, compose overrides this to development on build and run
+ARG NODE_ENV=production
+ENV NODE_ENV $NODE_ENV
 
 # Global npm deps in a non-root user directory
 ENV NPM_CONFIG_PREFIX=/app/.npm-global
 ENV PATH=$PATH:/app/.npm-global/bin
 
-ENV YARN_VERSION 1.19.1
-RUN yarn policies set-version "$YARN_VERSION"
+# Yarn
+ENV YARN_VERSION 1.22.22
+RUN yarn policies set-version ${YARN_VERSION}
```

Comment:
It uses yarn here, but npm later

Reply:
Switched completely to yarn in Dockerfile